sympy/core/add.py

```diff
@@ -5,20 +5,18 @@
 from .parameters import global_parameters
 from .logic import _fuzzy_group, fuzzy_or, fuzzy_not
 from .singleton import S
-from .operations import AssocOp
+from .operations import AssocOp, AssocOpDispatcher
 from .cache import cacheit
 from .numbers import ilcm, igcd
 from .expr import Expr
 
 # Key for sorting commutative args in canonical order
 _args_sortkey = cmp_to_key(Basic.compare)
 
-
 def _addsort(args):
     # in-place sorting of args
     args.sort(key=_args_sortkey)
 
-
```

Comment:
These blank lines are required by PEP8 I think - there's no reason to remove them in this PR.

Reply:
2 blank lines separation when there is no indentation, 1 blank line on indented code. Right?

Reply:
Yes, I think so